client/components/Events/index.js

```diff
@@ -25,10 +22,13 @@ class EventPage extends Component {
   // To get the clicked event and a recommended event
   componentDidMount(){
     window.scrollTo(0, 0);
-    this.props.getEvent();
+    this.props.getEvent(this.props.params.id);
+    this.props.getClub(this.props.params.club_id);
   }
 
-  joinThisClub(details={'club_id': 3, 'email': 'ig@uk.com'}){
+
+  joinThisClub(){
+    let details = {'club_id': this.props.club.id, 'email': this.props.user.app_user.email};
```

Comment:
No need to put the keys in quotes. You can just reference them as 
```
let details = { club_id: this.props.club.id..
```

Reply:
Ok... noted.